tests/core/test_status.py

```diff
@@ -20,7 +20,7 @@ async def test_async_status_success():
     assert st.success
 
 
-async def test_async_status_propagates_exception(failing_coroutine):
+async def test_async_status_propagates_exception(failing_coroutine, loop):
```

Comment:
Rather than adding a fixture which kills every task, I think it would be better to explicitly

``` python
try:
    # do stuff with status::AsyncStatus
finally:
    status.task.cancel()
```

for all these tests.